onnxruntime/core/providers/shared/node_unit/node_unit.cc

```diff
@@ -172,4 +238,54 @@ void NodeUnit::InitForNode() {
   }
 }
 
+void NodeUnit::InitForQDQGroup(const QDQ::NodeGroup& node_group) {
+  // Get inputs and outputs
+  GetQDQIODefs(target_node_, node_group, inputs_, true /* is_input */);
```

Comment:
>GetQDQIODefs [](http://example.com/codeflow?start=2&length=12)

if GetQDQIODefs() returns std::vector<NodeUnitIODef>, then we can just move these to the ctor member initializer list. would need to reorder the class data member declarations with inputs_ and outputs_ after target_node_